buildscripts/kokoro/unix.sh

```diff
@@ -88,8 +88,16 @@ if [[ -z "${ALL_ARTIFACTS:-}" ]]; then
   ./gradlew grpc-compiler:build grpc-compiler:publish $GRADLE_FLAGS \
     -Dorg.gradle.parallel=false -PrepositoryDir=$LOCAL_MVN_TEMP
 else
-  ./gradlew publish $GRADLE_FLAGS \
+  ./gradlew publish :grpc-core:versionFile $GRADLE_FLAGS \
```

Comment:
Why this is changed to only publish core publication?

Reply:
I _added_ to it. `publish` is there to export to the local maven repository. `versionFile` is there because we copy it from the `build/` directory. The point of having a task create the version file is to avoid _yet another_ place we have to manually change the version when doing a release.

Reply:
I mean, previously it publishes all publications, but now it has been changed to only publish core?

Reply:
My bad my bad. That was just running two tasks. Equivalent to

```
./gradlew publish
./gradlew :grpc-core:versionFile
```